### DISCUSSION TITLE
可以在cloudflare pages上部署吗

### DISCUSSION THREAD

**Discussion by darkelfzdy** [DIYgod/RSSHub#15506] [Q&A] [answered]
我fork了本项目，然后在cloudflare pages上进行了部署。已经部署成功，且设置了自己的域名，但不管是自己的域名还是cloudflare提供的域名，都无法登录rsshub的页面。不知道是否可以在cloudflare pages上部署

**Comment by DIYgod** [comment, marked as answer]
不可以在 Cloudflare Pages 上使用

**Comment by darkelfzdy** [comment]
太遗憾了
